docs/authorization/globus_app/apps.rst

```diff
@@ -0,0 +1,89 @@
+.. _globus_apps:
+
+.. currentmodule:: globus_sdk
+
+GlobusApps
+==========
+
+
+.. warning::
+
+    Currently ``GlobusApp`` only supports scripting-based use cases.
+
+    Hosted applications (i.e. web servers) should see :ref:`globus_authorizers` instead.
```

Comment:
I believe that a hosted webapp which uses 3-legged oauth to authenticate users can be developed on top of `GlobusApp` but requires that you implement your own login flow manager.

This is a case we haven't discussed much, but I haven't seen it as necessary to build explicit support, since it will always require custom login flow code.

But I think there's some blurring of lines happening (already in the current doc) here: a hosted client application and a resource server are different cases, and should be described as distinct.

Reply:
Web services that require dependent tokens effectively don't work with GlobusApp today.

Somebody probably could encapsulate that behavior in a login flow manager, but I would be very surprised if they did (and did it well). Given that we do have very plug-and-play interfaces for scripting cases I'm of the opinion that our messaging is "this works for scripting, not web services".

Reply:
> Web services that require dependent tokens effectively don't work with GlobusApp today.

That's not what I'm describing as what should work -- that's the resource server case.

I'm talking about a web application which is a pure client, and does the 3-legged OAuth login flow (authorization code grant). Think something like a heavyweight django app which is a client to Globus services.
That's a hosted web app but it's not a resource server.

Reply:
```suggestion
    Web services (and other hosted applications that act as resource servers) should see :ref:`globus_authorizers` instead.
```